app/routes/user.js

```diff
@@ -15,12 +15,15 @@ const { sendEmail } = require('../messaging/send-email')
 const { getHttpCodeFromResults } = require('../dto/mappers/bulk-requests')
 const { drop } = require('../cache')
 const { sendReportSomethingEmails, createAuditsForReportSomething } = require('../lib/email-helper')
+const { lookupPoliceForceByEmail } = require('../repos/police-forces')
+const { sendLoginToAudit } = require('../messaging/send-audit')
 
 module.exports = [
   {
     method: 'POST',
     path: '/user',
     options: {
+      auth: { scope: [scopes.admin] },
```

Comment:
I thought these had been added in a previous PR? Did I miss something?

Reply:
moved them to somewhere more visible